src/redux/reducers/siloDomainData/leads.js

```diff
@@ -245,16 +286,24 @@ const patchLead = (state, action) => {
     const settings = {
         leadPage: {
             [activeProject]: {
-                table: { $auto: {
-                    leads: { $autoArray: {
-                        $splice: [[tableIndex, 1, lead]],
-                    } },
-                } },
-                grid: { $auto: {
-                    leads: { $autoArray: {
-                        $splice: [[gridIndex, 1, lead]],
-                    } },
-                } },
+                table: {
+                    $auto: {
+                        leads: {
+                            $autoArray: {
+                                $splice: [[tableIndex, 1, lead]],
+                            },
+                        },
+                    },
+                },
+                grid: {
+                    $auto: {
```

Comment:
@ShrawanBK , was this necessary as well?

Reply:
That is due to the linter !!

Reply:
@ShrawanBK , could you talk with @tnagorra to resolve the linter issue on your setup?

Reply:
Why is it always @tnagorra

Reply:
I think the problem has been fixed.